### ISSUE TITLE
I'm wondering if this design follows Google's material design theme

### ISSUE THREAD

**Issue by mobeicanyue** [squidfunk/mkdocs-material#6340]
### Description

If the answer is yes, I suggest upgrade it from md2 to md3

https://m2.material.io/design
This theme looks like this style
![Screenshot_2023-11-16-17-05-59-647_com android chrome](https://github.com/squidfunk/mkdocs-material/assets/81098819/4f71bfde-2dd3-4e8a-993f-a13bb689d9d3)
![Screenshot_2023-11-16-17-06-02-415_com android chrome](https://github.com/squidfunk/mkdocs-material/assets/81098819/37029e3b-1e25-4886-b52c-50618195b19a)

And I think the following looks more beautiful
https://m3.material.io/
![Screenshot_2023-11-16-17-06-35-714_com android chrome](https://github.com/squidfunk/mkdocs-material/assets/81098819/9f21aad3-5450-4a24-9ee4-3b9a55f8315e)
![Screenshot_2023-11-16-17-06-38-431_com android chrome](https://github.com/squidfunk/mkdocs-material/assets/81098819/546a1225-c790-4369-a15a-829eaf190c16)

### Related links

- [Requesting a change](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/)
-

### Use Cases

https://m3.material.io/

### Before submitting

- [X] I have read and followed the [change request guidelines](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/).
- [X] I have verified that [my idea is a change request and not a bug report](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#its-not-a-bug-its-a-feature).
- [X] I have ensured that, to the best of my knowledge, [my idea will benefit the entire community](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#benefit-for-the-community).
- [X] I have included relevant links to [the documentation](https://squidfunk.github.io/mkdocs-material/), related [issues](https://github.com/squidfunk/mkdocs-material/issues), and [discussions](https://github.com/squidfunk/mkdocs-material/discussions) to underline the need for my idea.

**Comment by alexvoss** [comment]
Not sure about the degree of commitment to follow Google's lead. In any case, what you suggest sounds like a major undertaking that would potentially break a lot of things, especially carefully crafted customizations that people have come to depend on. There are also quite a few things I would prioritize well before such an update. With endless resources, we would give people a choice between M2 and M3 but we need to prioritize carefully. Just my 2p, I might be holding the wrong end of the stick.

**Comment by alexvoss** [comment]
Thanks for suggestion. I will convert this to a discussion so we can keep it around and continue it. I happen to know that some really like the design (and have said so recently rather than 10 years ago) but it is true that it does not reflect at least Google's current design preferences.

**Comment by alexvoss** [comment]
@squidfunk, my initial reaction was to convert this to a discussion but then it is a concrete enough suggestion, so could remain as an issue? What is the right approach? Close, put in our backlog and re-open when the time comes? Or convert to a discussion?

**Comment by squidfunk** [comment]
Thanks for asking! This is more like a question (as the title "I'm wondering ..." suggests) which is why I'm converting this to a discussion. I already answered this in https://github.com/squidfunk/mkdocs-material/discussions/5469#discussioncomment-5815087.

It's important to understand the history of the project: we started in 2016 with Material Design v1 and tried to follow the specs closely, but have since diverged at some points. We're currently not considering following the v2 and v3 specs closely, but we're planning on a face lift in the near future, where we will evaluate how we maybe can bring it a little closer to the current aesthetics again. This is, however, not a priority. We're currently trying to focus on functionality which is what almost all issues are about.
